integtest/src/main/java/org/corfudb/universe/cluster/process/ProcessCluster.java

```diff
@@ -0,0 +1,30 @@
+package org.corfudb.universe.cluster.process;
+
+import com.google.common.collect.ImmutableList;
+import org.corfudb.universe.cluster.Cluster;
+import org.corfudb.universe.cluster.ClusterException;
+import org.corfudb.universe.node.Node;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.corfudb.universe.node.Node'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)